thorlcr/thorutil/thbuf.cpp

```diff
@@ -1053,6 +1069,13 @@ class CCompressedSpillingRowStream: public CSimpleInterfaceOf<ISmartRowBuffer>,
     {
         return this;
     }
+    virtual unsigned __int64 getStatistic(StatisticKind kind) const
+    {
+        unsigned __int64 v = inactiveStats.queryStatistic(kind).get();
+        if (currentOutputIFileIO)
```

Comment:
this doesn't look thread safe.
An existing currentOutputIFileIO could be destroyed (in createNextOutputStream()) after this test.